yt/frontends/ramses/field_handlers.py

```diff
@@ -263,6 +264,18 @@ def offset(self):
         self._level_count = level_count
         return self._offset
 
+    @classmethod
+    def load_fields_from_yt_config(cls) -> Tuple[List[str], bool]:
```

Comment:
The way I read the function body, it looks like the bool should match `bool(fields)` (except in the case a section is found but no fields are specified, which to me would actually look like a bug ?)
I'd suggest simplifying the function so that it matches this signature (or alternatively give an explanation of why this wouldn't work)
```suggestion
    def load_fields_from_yt_config(cls) -> List[str]:
```